webrender/src/renderer.rs

```diff
@@ -3905,13 +3911,19 @@ impl<B: hal::Backend> Renderer<B> {
                 )
             });
 
+            // Unfortunatelly this clear rect must executed in a separate render pass
```

Comment:
let's reconsider this. What parts of the setup are different?

Reply:
Finaly managed to move this clear inside the next render pass in https://github.com/szeged/webrender/pull/343/commits/6eecba39c7ac073340f97ee8f3da138905f9647c